docs/modules/con-proxy-overview.adoc

```diff
@@ -146,9 +145,7 @@ filter.
 A one-to-many pattern, where one Virtual Cluster points to many Target Clusters (providing amalgamation),
 is not a supported use-case.
 
-
-
-=== Filter Chain
+=== Filter chain
```

Comment:
Now that `Virtual Cluster` and `Target Cluster` only have `==` this makes `FilterChain` a sub point of `Target Cluster` which is wrong.